### PR TITLE
New sample: CreateGeometries

### PR DESCRIPTION
**Pull request by dkemlage**
@nCastle1 Can you please code review this new sample: CreateGeoemtries?

### CODE REVIEW

File: src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/CreateGeometries.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
+// Copyright 2016 Esri.
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/CreateGeometries.cs]: Copyright date

File: src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/CreateGeometries.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+namespace ArcGISRuntime.Samples.CreateGeometries
+{
+    [Activity]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Create geometries",
+        "GeometryEngine",
+        "This sample demonstrates simple ways to create different types of Geometry. The geometries are added as Graphics to a GraphicsOverlay and displayed in a MapView. An Envelope is used to set the Viewpoint of the MapView.",
+        "",
+        "")]
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/CreateGeometries.cs]: In cases where there are no tags, the parameter should be left out.

File: src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Create geometries
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/README.md]: We should talk about the process for the readmes. Ideally we'd be using automation to create these (to save time and ensure things are consistent).

No action needed, just thinking out loud.

File: src/Forms/Shared/Samples/GeometryEngine/CreateGeometries/CreateGeometries.xaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"             
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
+             xmlns:mapping="clr-namespace:Esri.ArcGISRuntime.Mapping;assembly=Esri.ArcGISRuntime" 
+             x:Class="ArcGISRuntime.Samples.CreateGeometries.CreateGeometries">
+  <Grid>
+    <esriUI:MapView x:Name="MyMapView">
+      <mapping:Map/>
```
Comment (nCastle1) [inline src/Forms/Shared/Samples/GeometryEngine/CreateGeometries/CreateGeometries.xaml]: It looks like the map is later re-created in code. I think we can leave this out, and turn the mapview declaration into one line, `<esriUI:MapView x:Name="MyMapView" />`.

File: src/Forms/Shared/Samples/GeometryEngine/CreateGeometries/CreateGeometries.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+using Xamarin.Forms;
+
+namespace ArcGISRuntime.Samples.CreateGeometries
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Create geometries",
+        "GeometryEngine",
+        "This sample demonstrates simple ways to create different types of Geometry. The geometries are added as Graphics to a GraphicsOverlay and displayed in a MapView. An Envelope is used to set the Viewpoint of the MapView.",
+        "",
+        "")]
```
Comment (nCastle1) [inline src/Forms/Shared/Samples/GeometryEngine/CreateGeometries/CreateGeometries.xaml.cs]: Same comment about tags as above, I think this applies to all of the platforms, so I won't comment again.

**Comment by dkemlage** [comment]
The sample was merged into v.next and master.

**Pull request merged**